hedera-node/src/main/java/com/hedera/services/context/BasicTransactionContext.java

```diff
@@ -223,10 +231,15 @@ public ResponseCodeEnum status() {
 	}
 
 	@Override
-	public TxnAccessor accessor() {
+	public SignedTxnAccessor accessor() {
```

Comment:
We shouldn't need to use the implementation type here either.

Reply:
Fixed!!